.circleci/config.yml

```diff
@@ -180,8 +180,9 @@ jobs:
       - run:
           name: Install Dependencies
           command: |
-            sudo apt-get install nuget
-            nuget install Grpc.Tools -Version 1.14.1 -OutputDirectory protobuf-definitions/
+            sudo apt-get install nuget=2.12
```

Comment:
Probably need to update the directions in https://github.com/Unity-Technologies/ml-agents/blob/master/protobuf-definitions/README.md#set-up--installation too? Maybe just direct those to here?

Reply:
yeah, I'll fix up the docs when I split the PRs.